src/common/utils/tracking.ts

```diff
@@ -0,0 +1,24 @@
+export enum EventCategory {
+  COMPARE = 'compare',
+  EXPLORE = 'explore',
+  ENGAGEMENT = 'engagement',
+}
+
+export enum EventAction {
+  SHARE = 'share',
+  SAVE_IMAGE = 'save image',
+  COPY_LINK = 'copy link',
+}
+
+export default function trackEvent(
+  category: EventCategory,
+  action: EventAction,
+  label: string,
+  value?: number,
+) {
+  window.gtag('event', action, {
```

Comment:
Would it be worth using the official react GA plugin? https://github.com/react-ga/react-ga

Reply:
Oh that's pretty neat, thanks, will give it a try!

Reply:
actually I will do that later - I want to debug the issues that Igor is mentioning (page views might be wrong) without the extra layer